app/assets/javascripts/jquery.textarea_autosize.js

```diff
@@ -0,0 +1,54 @@
+/*!
```

Comment:
I think this file should live in `vendor/assets`, you'll need to explicitly require it in application.js though if I recall correctly.